streams/examples/src/test/java/org/apache/kafka/streams/examples/wordcount/WordCountTransformerTest.java

```diff
@@ -45,11 +50,26 @@ public void test() {
                 .withLoggingDisabled() // Changelog is not supported by MockProcessorContext.
                 // Caching is disabled by default, but FYI: caching is also not supported by MockProcessorContext.
                 .build();
-            store.init(context, store);
-            context.register(store, null);
+            store.init(context.getStateStoreContext(), store);
+            context.getStateStoreContext().register(store, null);
         }
         final Transformer<String, String, KeyValue<String, String>> transformer = supplier.get();
-        transformer.init(context);
+        transformer.init(new org.apache.kafka.streams.processor.MockProcessorContext() {
+            @Override
+            public <S extends StateStore> S getStateStore(final String name) {
+                return context.getStateStore(name);
+            }
+
+            @Override
+            public <K, V> void forward(final K key, final V value) {
+                context.forward(new Record<>(key.toString(), value.toString(), 0L));
```

Comment:
Why `toString()` -- should we not assume that both `K` and `V` are of type `String`? I guess we cannot declare the input parameters as `String`, but wondering if we should rather cast here?

Reply:
Yes, both K and V are types of String here, we should use cast here.